tests/migrations/test_migrations_recorded_atomically/0001_initial.py

```diff
@@ -0,0 +1,23 @@
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    operations = [
+        migrations.CreateModel(
+            "Author2",
+            [
+                ("id", models.AutoField(primary_key=True)),
+                ("name", models.CharField(max_length=255)),
+                ("slug", models.SlugField(null=True)),
+                ("age", models.IntegerField(default=0)),
+                ("silly_field", models.BooleanField(default=False)),
+            ],
+        ),
+        migrations.AlterUniqueTogether(
+            name='Author2',
+            unique_together={('name', 'slug')},
+        )
```

Comment:
No need to have such complex operations for a minimal test case; it only adds noise that future reader could interpret as necessary to cover the changes. A simple `CreateModel` with an `id` field should be enough

```python
migrations.CreateModel(
    "Foo",
    [
        ("id", models.AutoField(primary_key=True)),
    ],
)

Reply:
Fixed.